cmd/maestro/main.go

```diff
@@ -36,11 +37,29 @@ func main() {
 			klog.Fatalf("can't parse log level: %v", err)
 		}
 
-		// Initialize zap logger
-		zc := zap.NewDevelopmentConfig()
+		// Initialize zap logger based on environment
+		var zc zap.Config
```

Comment:
for zap, it can support dynamically update the log level. that can help us on the troubleshooting. Can we do it instead of update the level in environment variable only? maybe define a configmap for the log level.